corehq/apps/app_manager/tests/data/suite/search_command_detail.xml

```diff
@@ -16,6 +16,11 @@
           <xpath function="case_name"/>
         </text>
       </template>
+      <sort direction="ascending" order="1" type="string">
```

Comment:
@orangejenny can you please help confirm if this is because of https://github.com/dimagi/commcare-hq/pull/29784/files?

I checked out to master, and added
```
# wrap to have assign_references called
self.app = Application.wrap(self.app.to_json())
# reset to newly wrapped module
self.module = self.app.modules[0]
```
in `setUp`, that gave me two sort blocks in the diff which makes me think that wrapping the application doc brings sort setting into action, though here there was a diff of 4, which is not clear why that would be.

Reply:
found the real reason on why this was happening here, fixing it here https://github.com/dimagi/commcare-hq/pull/29866.

Its expected to have these 4 sort blocks added after wrap